client/src/App.js

```diff
@@ -32,7 +33,7 @@ function App() {
         <Route
           path="/campaigns"
           element={
-            <h1>campaigns</h1>
+            <OneCampaign id={2} />
```

Comment:
i think in the campaign itself you will make a get request to /api/campagin/${id}